exercises/nucleotide-count/src/test/java/NucleotideCounterTest.java

```diff
@@ -13,18 +13,11 @@
     @Rule
     public ExpectedException expectedException = ExpectedException.none();
 
-    @Test
-    public void testEmptyDnaStringHasNoAdenine() {
-        NucleotideCounter nucleotideCounter = new NucleotideCounter("");
-        assertThat(nucleotideCounter.count('A'), is(0));
-    }
-
     @Ignore("Remove to run test")
     @Test
     public void testEmptyDnaStringHasNoNucleotides() {
         NucleotideCounter nucleotideCounter = new NucleotideCounter("");
         Map<Character, Integer> counts = nucleotideCounter.nucleotideCounts();
-        assertThat(counts.size(), is(4));
```

Comment:
This size test hasn't ever been part of the canonical data, but I am wondering if it was in here for a more Java-specific reason...

@exercism/java are we good to omit this? :slightly_smiling_face:

Reply:
The size() of counts is always 4 due to the .emptyCount() function, so I preferred to remove it from most places, however testing it in some places

Reply:
If it's not part of the canonical data then I think we should remove it :)